scripts/process_logs.yml

```diff
@@ -0,0 +1,53 @@
+
+input_logs:
+  - path: .                        # Where to look for log files
+    recursive: yes                 # Whether to descend into subdirectories recursively
+    pattern: Node(\d+)\.log.*      # Log file regex pattern to look for
+    node_group: 1                  # Group number that matches node identifier
+    only_timestamped: no           # Whether to discard non-timestamped lines
+
+
+chains:
```

Comment:
We need documentation and description how to use it and what each parameter means here.